hydra-explorer/src/Hydra/Explorer.hs

```diff
@@ -82,26 +84,33 @@ readModelGetHeadIds = readTVarIO
 main :: IO ()
 main = do
   withTracer (Verbose "hydra-explorer") $ \tracer -> do
+    opts <- execParser hydraExplorerOptions
+    let Options
```

Comment:
These could be part of the `where` block as per our coding conventions :trollface:

Reply:
i dont see a way... opts is the result obtained from an action.

unless i am not understanding ur request.
can u plz elaborate more 🙏

Reply:
Oh yeah, you are right